modules/kafka/src/test/java/org/testcontainers/containers/KafkaContainerTest.java

```diff
@@ -195,13 +195,7 @@ public void testUsageWithListener() throws Exception {
                 .withNetwork(network);
             // }
             // createKCatContainer {
-            GenericContainer<?> kcat = new GenericContainer<>("confluentinc/cp-kcat:7.9.0")
-                .withCreateContainerCmdModifier(cmd -> {
-                    cmd.withEntrypoint("sh");
-                })
-                .withCopyToContainer(Transferable.of("Message produced by kcat"), "/data/msgs.txt")
-                .withNetwork(network)
-                .withCommand("-c", "tail -f /dev/null")
+            KCatContainer kcat = new KCatContainer().withNetwork(network)
```

Comment:
I just noticed that this is part of the documentation. So, we should keep it as is.